cms/djangoapps/contentstore/views/tests/test_import_export.py

```diff
@@ -807,3 +807,7 @@ def test_library_content_on_course_export_import(self, publish_item):
             self.source_course.location,
             dest_course.location
         )
+
+    def test_branch_settings(self):
```

Comment:
In particular, https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/modulestore/tests/test_semantics.py would be a good file.